docs/notebooks/visualizing_with_tensorboard.pct.py

```diff
@@ -153,17 +153,18 @@ def acquire(self, search_space, models, datasets=None):
 # %% [markdown]
 # ## Using Tensorboard with Ask-Tell Optimization
 #
-# To use Tensorboard logging with the [Ask-Tell interface](ask_tell_optimization.ipynb), you must also explicitly set the optimization step number before calling ask or tell:
+# To use Tensorboard logging with the [Ask-Tell interface](ask_tell_optimization.ipynb), you must also explicitly set the optimization step number before calling ask or tell methods. If fitting the model on initialization of the Ask-Tell interface (default), then you must also set the optimization step number before initializing the interface to log initial model fitting as a different step.
 
 # %%
 summary_writer = tf.summary.create_file_writer("logs/tensorboard/experiment4")
 trieste.logging.set_tensorboard_writer(summary_writer)
 
+trieste.logging.set_step_number(0)
```

Comment:
(step number does deault to 0, so strictly speaking this isn't necessary, but probably best to leave it in)